pkg/minikube/cluster/cluster.go

```diff
@@ -247,23 +252,30 @@ func SetupCerts(d drivers.Driver) error {
 		return errors.Wrap(err, "Error generating certs")
 	}
 
-	client, err := sshutil.NewSSHClient(d)
-	if err != nil {
-		return errors.Wrap(err, "Error creating new ssh client")
-	}
+	copyableFiles := []assets.CopyableFile{}
```

Comment:
Any reason to keep two lists of files to copy here? Could we append the certs to the same list from above?